docs/ai/quickstart.md

```diff
@@ -0,0 +1,170 @@
+# Quickstart Guide
+
+Welcome to the **Unity Catalog AI Core Library**! This guide will help you get started with installing the library, setting up your environment, and creating and executing Unity Catalog (UC) functions. Whether you're using your own UC open-source server or connecting with a Databricks-managed UC, this quickstart will walk you through the essential steps.
+
+---
+
+## Installation
+
+Install the Unity Catalog AI Core library directly from PyPI:
+
+```sh
+pip install unitycatalog-ai
+```
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- **Python Version**: Python 3.10 or higher is recommended for simplified type hint processing within the Unity Catalog functions client.
+
+### Unity Catalog Open Source
+
+If you're looking to use an Open Source Unity Catalog service, ensure that you have met all of the prerequisites and have followed the
+[Unity Catalog Quickstart Guide](../quickstart.md). Once your server is up and running and you are able to create a catalog, schemas, and functions, you're all set to continue with this guide.
+
+### Databricks Unity Catalog
+
+If you are going to interface with ``Databricks Unity Catalog``, ensure that you have installed the ``Databricks SDK`` to enable access
+to your Databricks Unity Catalog service.
+
+```sh
+pip install databricks-sdk
```

Comment:
```suggestion
pip install unitycatalog-ai[databricks]
```
we could mention what extra packages are installed by this, but using `[databricks]` can save us some maintenance work in case we update the dependencies in the future.